tests/components/influxdb/test_init.py

```diff
@@ -62,8 +62,12 @@ def mock_client_fixture(request):
 def get_mock_call_fixture(request):
     """Get version specific lambda to make write API call mock."""
     if request.param == influxdb.API_VERSION_2:
-        return lambda body, precision=None: call(
-            bucket=DEFAULT_BUCKET, record=body, write_precision=precision
+        return (
+            lambda body, precision=None: call(
+                bucket=DEFAULT_BUCKET, record=body, write_precision=precision
+            )
+            if precision is not None
+            else call(bucket=DEFAULT_BUCKET, record=body)
```

Comment:
Can we make the same pattern as in the code under test?

Even better would be to use pytest parametrization for the kwargs.

Reply:
Something like this? I don't know how to address it in the second way you are proposing.

Reply:
If we would use pytest.parametrize we could possibly remove the if/else check and just forward the keyword arguments as parameters in the v2_call. It's a larger refactor I think, so we don't need to do it here.